pkg/controller/elasticsearch/client/client.go

```diff
@@ -85,6 +85,8 @@ type Client interface {
 	GetNodesStats(ctx context.Context) (NodesStats, error)
 	// GetLicense returns the currently applied license. Can be empty.
 	GetLicense(ctx context.Context) (License, error)
+	// StartTrial starts a 30-day trial license.
```

Comment:
"StartTrial starts a 30-day trial period" (which gives you access to platinum features)